session/session.go

```diff
@@ -1183,6 +1193,11 @@ func (s *session) CachedPlanExec(ctx context.Context,
 	if err != nil {
 		return nil, err
 	}
+	// after drop unique index like operation, the former cached plan is
+	// not point plan and will be set to nil in `optimizePrepareStatement` func
+	if prepared.CachedPlan == nil {
```

Comment:
We should not call `CachePlanExec` if `CachedPlan` is nil.

Reply:
add schema check in `IsCachedExecOk`